packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -59,22 +65,36 @@ const MediaReplaceFlow = ( {
 		}
 	};
 
+	const onError = ( message ) => {
+		const errorElement = document.createElement( 'div' );
+		errorElement.innerHTML = renderToString( message );
+		const renderMsg =
+			errorElement.textContent || errorElement.innerText || '';
```

Comment:
What's the goal with this code?

Reply:
The default error contains some HTML that, for example, makes the filename bold. The notice, by default, accepts strings only and so I am removing the html from the error.

Reply:
Ah, ok. It doesn't seem to do that when I tested it in the browser console.

`renderToString` basically escapes the content, so `innerHTML` treats it as plain text. 

Are there other examples of this in the codebase?

Reply:
I explained poorly! `message` is a React Element and so `renderToString` returns DOM   elements which work OK with `innerHTML`.

Reply:
Oh, I see 👍